src/pages/billing/Project.js

```diff
@@ -221,10 +224,42 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     _.map('workspace', workspaces)
   ), [billingProject, workspaces])
 
+  const spendReportKey = 'spend report'
+  const maybeLoadTotalCost = reportErrorAndRethrow('Unable to retrieve spend report data')(async () => {
+    if (!updatingTotalCost && totalCost === null && tab === spendReportKey) {
+      setUpdatingTotalCost(true)
+      const endDate = new Date().toISOString().slice(0, 10)
+      const startDate = new Date((Date.now() - spendReportLengthInDays * 24 * 60 * 60 * 1000)).toISOString().slice(0, 10)
+      const spend = await Ajax(signal).Billing.getSpendReport({ billingProjectName: billingProject.projectName, startDate, endDate })
```

Comment:
Would it be possible to display a spinning wheel while we're waiting for this to return? Even though it only appeared for a moment, I thought something was wrong when I opened the spend report and saw underscores.
![Screen Shot 2022-03-02 at 5 27 49 PM](https://user-images.githubusercontent.com/19961550/156460597-ffd95038-5462-4c5e-af8f-73ce9f4d065b.png)

Reply:
I created https://broadworkbench.atlassian.net/browse/WOR-171 for us to decide what we want to show to the user while the report is generating. This particular page already has 2 spinning wheels in play when the page first loads, and I'm somewhat reluctant to introduce a third.